server/repository/src/db_diesel/name_insurance_join_row.rs

```diff
@@ -1,12 +1,29 @@
 use super::{
-    ChangeLogInsertRow, ChangelogRepository, ChangelogTableName, RowActionType, StorageConnection,
+    ChangeLogInsertRow, ChangelogRepository, ChangelogTableName, EqualFilter, RowActionType,
+    StorageConnection,
 };
 
-use crate::{repository_error::RepositoryError, Upsert};
+use crate::{
+    diesel_macros::{apply_equal_filter, apply_sort},
+    repository_error::RepositoryError,
+    Sort, Upsert,
+};
 use diesel::prelude::*;
 use diesel_derive_enum::DbEnum;
 use serde::{Deserialize, Serialize};
 
+#[derive(Clone, Default, PartialEq, Debug)]
+pub struct NameInsuranceJoinFilter {
```

Comment:
I don't see any need for this filter in the frontend, as far as I know we don't need to filter in that view other than by the patient name which we're passing in.
I'd be tempted to just remove the filter entirely....

Reply:
I was thinking about too, doesn't seem like we need it. Will remove it!